### DISCUSSION TITLE
Serverpod mini with Cloud Run & Firestore

### DISCUSSION THREAD

**Discussion by FXschwartz** [serverpod/serverpod#2786] [Q&A] [answered]
Haven't seen any documentation on this and just want to confirm. I'd like to deploy a small dart backend to Cloud Run that communicates with the rest of my Firebase services including Cloud Firestore.

I saw that Serverpod mini does not need to be connected to Postgres but will it work with other DB's like Cloud Firestore?

Also does Serverpod mini have to have the `myminipod_flutter` directory or can it be ignored?

**Comment by Isakdl** [comment, marked as answer]
It should work, but you'll need to add the Cloud Firestore integration yourself as that is not preinstalled. 

The flutter folder can safely be deleted, it is just created as an example/starting point in the template.
